entrypoint.sh

```diff
@@ -256,7 +258,13 @@ xEOF
         echo "      - ${NETWORK}" >> ${COMPOSE_FILE}
       done
       for NETWORK in ${LAUNCH_EXT_NETWORKS}; do
-        echo "      - ${NETWORK}" >> ${COMPOSE_FILE}
+        echo "      ${NETWORK}:" >> ${COMPOSE_FILE}
+        if [ -n "${LAUNCH_IPV4_ADDRESS}" ]; then
+          echo "        ipv4_address: ${LAUNCH_IPV4_ADDRESS}" >> ${COMPOSE_FILE}
```

Comment:
Now I realize - the same IP address will be set for *all* the `LAUNCH_EXT_NETWORKS`. This must be handled differently.